File: logic build problem/easy problems/python/isPalindromeNumber.py
def is_palindromeNumber(n:int)->bool:
    if len(str(n)) == 1:
        return True
    else:
        number = str(abs(n))
        length = len(number)
        half = length // 2
        for i in range(half):
           if number[i] != number[length - i - 1]:
               return False
        return True

print(is_palindromeNumber(12321))
            
        

File: logic build problem/easy problems/Typescript/isPalindromeNumber.ts
function is_palindromeNumber(n:number):boolean{
    let s:string = Math.abs(n).toString()
    let len:number = s.length
    let half:number = Math.floor(len/2)
    for(let i:number = 0;i<half;i++){
        if(s[i] != s[len - i -1]){
            return false
        }
    }
    return true
}
console.log(is_palindromeNumber(12321))
